languages/c/ast/ast_c_build.ml

```diff
@@ -77,7 +77,8 @@ let rec ifdef_skipper xs f =
       | Some ifdef -> (
           match ifdef with
           | Ifdef tok -> (
-              logger#info "skipping: %s" (Tok.content_of_tok tok);
+              Logs.info (fun m ->
```

Comment:
Like I said in Slack, most of our logger#info should be converted in Logs.debug,
because Logs.info can be seen by regular users when using --verbose.

Reply:
I'll refine the recommendations in `Logs_.mli` as they become clearer to us.